### PR TITLE
Add pencil icon to edit property name

### PR DESCRIPTION
**Pull request by beyond2void**

### CODE REVIEW

**Comment by beyond2void** [comment]
still one thing in progress: avoid it navigating inside

File: frontend/config-assistant/src/components/gui-editor/PropertyMetadata.vue [hunk 1]
```diff
@@ -145,6 +148,17 @@ function getDisplayNameOfNode(node: GuiEditorTreeNode): string {
 function isInvalid(): boolean {
   return !props.validationResults.valid;
 }
+
+function focusOnPropertyLabel(): void {
+  const id: string = getId();
```
Comment (PaulBredl) [inline frontend/config-assistant/src/components/gui-editor/PropertyMetadata.vue:153]: For these three lines and "element.focus" there exists focusUtils, which you could use I think

**Review by PaulBredl — APPROVED** [review]

**Review by PaulBredl — CHANGES_REQUESTED** [review]
Still todo: clicking on the property name should zoom into it

**Review by PaulBredl — CHANGES_REQUESTED** [review]
When I add a new (additional) property, the name of the property is no longer selected and I can no longer enter anything

**Review by PaulBredl — APPROVED** [review]

**Review by Logende — APPROVED** [review]

**Pull request merged**